Dev4Agriculture.ISO11783.ISOXML/ISOExtensions/ISOPartField.cs

```diff
@@ -63,7 +63,47 @@ public bool IsInField(decimal longitude,decimal latitude)
         {
             return PolygonnonTreatmentZoneonly.First().IsInPolygon(longitude, latitude);
         }
-        
+
+        public IntersectionResult TryGetOverlapWithPartfield(ISOPartfield secondField)
```

Comment:
Please add a function description with /// that describes the several steps and how to interpret the result